Resources/Prototypes/Catalog/Fills/Boxes/ammunition.yml

```diff
@@ -349,97 +349,104 @@
 # Shotgun Shells
 - type: entity
   name: box of shotgun beanbag cartridges
-  parent: BoxAmmoProvider
+  parent: BoxMagazine
   id: BoxBeanbag
   description: A box full of beanbag shots, designed for riot shotguns.
   components:
-    - type: BallisticAmmoProvider
-      proto: ShellShotgunBeanbag
-      capacity: 12
+    - type: StorageFill
+    contents:
+      - id: ShellShotgunBeanbag
+        amount: 12
```

Comment:
You should fix the bug with the visuals rather than having it spawn all 12 entities at once.

Reply:
Cartridges are extremely frequent and it's not feasible to have every single ammo box spawn all of its entities.

Reply:
Okay, I'll try to remove the inventory

Reply:
done, now it works as a store, and there is no inventory through the action menu